doc/sphinx/source/community/maintainer.rst

```diff
@@ -0,0 +1,25 @@
+Maintaining a recipe
+====================
+
+As development of the ESMValTool continues, new features may be added, old ones replaced or retired or
+the interface of library functions may change when updating to new versions. This or for example the
+withdrawl of datasets used by a recipe can result in an existing recipe to stop workinge. Such "broken"
+recipes might require some work to fix such problems and make the recipe fully functional again.
+
+A first **contact point** for the technical lead development team in such cases is the recipe "maintainer". The recipe
+maintainer is then asked to check the affected recipe and if possible, fix the problems or work with the technical
+lead development team to find a solution. Ideally, a recipe maintainer is able to tell whether the results of a fixed
+recipe are scientifically valid and look as expected. Being a recipe maintainer constists of the following tasks:
+
+* answering timely to requests from the technical lead deveopment team, e.g. if a recipe is broken
```

Comment:
```suggestion
* answering timely to requests from the technical lead development team, e.g. if a recipe is broken
```
also unsure if "timely" does anything here - Just saying timely sounds too vague and then I'd probably just leave it out.